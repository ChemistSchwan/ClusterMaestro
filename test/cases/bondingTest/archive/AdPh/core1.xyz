26
Properties=species:S:1:pos:R:3:tags:I:1 pbc="F F F"
C       1.07573747       0.69576201       4.19532208        0 
C      -0.26372455       0.27200459       4.81764897        1 
C      -1.30631352       1.35122178       4.49712722        9 
C      -0.73335002      -1.05199111       4.19513594        8 
C       0.92894555       0.86762289       2.67853999        7 
C       0.44990576      -0.46639674       2.09328051        6 
C      -0.89990223      -0.89922013       2.67835150        5 
C      -1.93108409       0.19693768       2.38678173        4 
C      -1.48331691       1.53455830       2.98490771        3 
C      -0.13099707       1.93596400       2.38691879        2 
H       1.40234997       1.63148091       4.65359747       18 
H       1.84473526      -0.05208441       4.39620063       19 
H      -0.98348761       2.29821617       4.93507756       20 
H      -2.26390446       1.06119557       4.93497850       21 
H      -1.67983269      -1.34616913       4.65326529       16 
H      -0.01250342      -1.84635122       4.39604166       17 
H       1.19205481      -1.23458332       2.32066014       14 
H       0.35501820      -0.36811874       1.00921399       15 
H      -2.05667340       0.27675801       1.30734622       12 
H      -2.90166738      -0.06721962       2.80852982       13 
H      -0.20633076       2.06426232       1.30747517       10 
H       0.16641597       2.89685294       2.80871108       11 
H      -0.14623458       0.15027994       5.88226089        0 
H       1.87075854       1.16361708       2.24564848        0 
H      -1.22812224      -1.83021745       2.24529845        0 
H      -2.21730685       2.29433352       2.77038566        0 
