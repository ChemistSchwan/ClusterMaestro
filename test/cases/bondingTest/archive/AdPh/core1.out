      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. MÃ¤rz, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/09/25 at 10:01:15.521     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : xtb start.xyz -P 1
          coordinate file            : start.xyz
          omp threads                :                     1
          number of atoms            :                    26
          number of electrons        :                    56
          charge                     :                     0
          spin                       :                   0.0
          first test random number   :      0.96308652724249

########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  56          :
          :  # atomic orbitals                  56          :
          :  # shells                           36          :
          :  # electrons                        56          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -29.9974930 -0.299975E+02  0.228E+00   12.75       0.0  T
   2    -30.0662533 -0.687603E-01  0.138E+00   12.68       1.0  T
   3    -30.0669678 -0.714482E-03  0.747E-01   12.65       1.0  T
   4    -30.0671256 -0.157819E-03  0.100E-01   12.63       1.0  T
   5    -30.0671908 -0.651600E-04  0.478E-02   12.62       1.0  T
   6    -30.0671913 -0.503434E-06  0.204E-03   12.63       9.6  T
   7    -30.0671913 -0.123773E-07  0.983E-04   12.63      20.0  T
   8    -30.0671913 -0.915641E-09  0.190E-04   12.63     103.4  T

   *** convergence criteria satisfied after 8 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6396839             -17.4067
       ...           ...                  ...                  ...
        22        2.0000           -0.4272291             -11.6255
        23        2.0000           -0.4270891             -11.6217
        24        2.0000           -0.4257178             -11.5844
        25        2.0000           -0.4252925             -11.5728
        26        2.0000           -0.3971670             -10.8075
        27        2.0000           -0.3967672             -10.7966
        28        2.0000           -0.3953216             -10.7572 (HOMO)
        29                          0.0686393               1.8678 (LUMO)
        30                          0.0715782               1.9477
        31                          0.0719768               1.9586
        32                          0.0882430               2.4012
        33                          0.0997164               2.7134
       ...                                ...                  ...
        56                          0.3230197               8.7898
      -------------------------------------------------------------
                  HL-Gap            0.4639609 Eh           12.6250 eV
             Fermi-level           -0.1633412 Eh           -4.4447 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.047 sec
 SCC setup                      ...        0 min,  0.000 sec (  1.040%)
 Dispersion                     ...        0 min,  0.000 sec (  0.694%)
 integral evaluation            ...        0 min,  0.017 sec ( 35.479%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  0.806%)
 iterations                     ...        0 min,  0.015 sec ( 32.204%)
 molecular gradient             ...        0 min,  0.014 sec ( 29.412%)
 printout                       ...        0 min,  0.000 sec (  0.255%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -29.636603005286 Eh    ::
         :: gradient norm              0.053890181270 Eh/a0 ::
         :: HOMO-LUMO gap             12.625019366499 eV    ::
         ::.................................................::
         :: SCC energy               -30.067191307256 Eh    ::
         :: -> isotropic ES            0.003261728119 Eh    ::
         :: -> anisotropic ES          0.006336974074 Eh    ::
         :: -> anisotropic XC          0.014437371927 Eh    ::
         :: -> dispersion             -0.023986242701 Eh    ::
         :: repulsion energy           0.430403483566 Eh    ::
         :: add. restraining           0.000000000000 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.6396839             -17.4067
       ...           ...                  ...                  ...
        16        2.0000           -0.4814159             -13.1000
        17        2.0000           -0.4810340             -13.0896
        18        2.0000           -0.4664257             -12.6921
        19        2.0000           -0.4660833             -12.6828
        20        2.0000           -0.4655384             -12.6679
        21        2.0000           -0.4295660             -11.6891
        22        2.0000           -0.4272291             -11.6255
        23        2.0000           -0.4270891             -11.6217
        24        2.0000           -0.4257178             -11.5844
        25        2.0000           -0.4252925             -11.5728
        26        2.0000           -0.3971670             -10.8075
        27        2.0000           -0.3967672             -10.7966
        28        2.0000           -0.3953216             -10.7572 (HOMO)
        29                          0.0686393               1.8678 (LUMO)
        30                          0.0715782               1.9477
        31                          0.0719768               1.9586
        32                          0.0882430               2.4012
        33                          0.0997164               2.7134
        34                          0.1008098               2.7432
        35                          0.1106553               3.0111
        36                          0.1140152               3.1025
        37                          0.1153201               3.1380
        38                          0.1326226               3.6088
        39                          0.1475694               4.0156
       ...                                ...                  ...
        56                          0.3230197               8.7898
      -------------------------------------------------------------
                  HL-Gap            0.4639609 Eh           12.6250 eV
             Fermi-level           -0.1633412 Eh           -4.4447 eV

     #   Z        covCN         q      C6AA      Î±(0)
     1   6 C      3.814    -0.059    21.624     6.619
     2   6 C      3.856    -0.014    20.785     6.484
     3   6 C      3.816    -0.059    21.623     6.618
     4   6 C      3.814    -0.059    21.624     6.619
     5   6 C      3.856    -0.014    20.781     6.483
     6   6 C      3.817    -0.059    21.623     6.618
     7   6 C      3.856    -0.014    20.781     6.483
     8   6 C      3.815    -0.059    21.624     6.618
     9   6 C      3.857    -0.013    20.777     6.483
    10   6 C      3.815    -0.059    21.624     6.618
    11   1 H      0.924     0.025     2.653     2.547
    12   1 H      0.924     0.025     2.655     2.548
    13   1 H      0.924     0.027     2.628     2.535
    14   1 H      0.924     0.027     2.628     2.535
    15   1 H      0.924     0.025     2.653     2.547
    16   1 H      0.924     0.025     2.655     2.548
    17   1 H      0.924     0.027     2.623     2.533
    18   1 H      0.924     0.026     2.638     2.540
    19   1 H      0.925     0.026     2.650     2.546
    20   1 H      0.924     0.025     2.654     2.547
    21   1 H      0.925     0.026     2.650     2.546
    22   1 H      0.924     0.025     2.654     2.547
    23   1 H      0.926     0.025     2.665     2.553
    24   1 H      0.926     0.024     2.669     2.555
    25   1 H      0.926     0.024     2.669     2.555
    26   1 H      0.926     0.024     2.668     2.554

 Mol. C6AA /auÂ·bohrâ¶  :       5195.283688
 Mol. C8AA /auÂ·bohrâ¸  :     114417.014736
 Mol. Î±(0) /au        :        106.380982


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  C   3.998        C    5 0.998    C    2 0.998    H   12 0.977    H   11 0.977
     2  C   3.997        C    3 0.998    C    1 0.998    C    4 0.998    H   23 0.970
     3  C   3.998        C    2 0.998    C    9 0.997    H   14 0.977    H   13 0.977
     4  C   3.998        C    7 0.998    C    2 0.998    H   16 0.977    H   15 0.977
     5  C   3.997        C    1 0.998    C   10 0.998    C    6 0.997    H   24 0.970
     6  C   3.998        C    5 0.997    C    7 0.997    H   18 0.977    H   17 0.976
     7  C   3.997        C    4 0.998    C    8 0.998    C    6 0.997    H   25 0.970
     8  C   3.998        C    9 0.998    C    7 0.998    H   19 0.977    H   20 0.977
     9  C   3.997        C    8 0.998    C   10 0.998    C    3 0.997    H   26 0.970
    10  C   3.998        C    9 0.998    C    5 0.998    H   21 0.977    H   22 0.977
    11  H   0.999        C    1 0.977
    12  H   0.999        C    1 0.977
    13  H   0.999        C    3 0.977
    14  H   0.999        C    3 0.977
    15  H   0.999        C    4 0.977
    16  H   0.999        C    4 0.977
    17  H   0.999        C    6 0.976
    18  H   0.999        C    6 0.977
    19  H   0.999        C    8 0.977
    20  H   0.999        C    8 0.977
    21  H   0.999        C   10 0.977
    22  H   0.999        C   10 0.977
    23  H   0.999        C    2 0.970
    24  H   0.999        C    5 0.970
    25  H   0.999        C    7 0.970
    26  H   0.999        C    9 0.970

molecular dipole:
                 x           y           z       tot (Debye)
 q only:       -0.001       0.001       0.001
   full:       -0.000       0.000      -0.000       0.002
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:       -0.011      -0.043      -0.009      -0.060       0.062       0.020
  q+dip:        0.027       0.075       0.022       0.048      -0.049      -0.049
   full:        0.016       0.038       0.013       0.021      -0.022      -0.030


           -------------------------------------------------
          | TOTAL ENERGY              -29.636603005286 Eh   |
          | GRADIENT NORM               0.053890181270 Eh/Î± |
          | HOMO-LUMO GAP              12.625019366499 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/09/25 at 10:01:15.589     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.067 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.067 sec
 * ratio c/w:     0.995 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.048 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.047 sec
 * ratio c/w:     0.997 speedup

